### PR TITLE
Fix inconsistently poorly formatted README.md

### PR DESCRIPTION
**Pull request by Sean-Der**
Please fix your spelling issues in the commit before this can be merged

### CODE REVIEW

**Comment by Sean-Der** [comment]
@kc5nra please merge

**Comment by Sean-Der** [comment]
FIXED

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ sporkel [![Build Status](https://travis-ci.org/kc5nra/sporkel.svg?branch=travis-
 =======
 
 
-           _,,,,,,,_                                                    
+           _,,,,,,,_
```
Comment (Sean-Der) [inline README.md]: Nice work @Sean-Der

**Comment by kc5nra** [comment]
"Fix inconsistently poorly formatted" doesn't make sense, please make a commit title that addresses the issues succinctly and with proper English.

**Comment by kc5nra** [comment]
Commit title exceeds the maximum 50 characters

**Comment by kc5nra** [comment]
If it was pristine, it wouldn't need a minor addition

**Comment by Sean-Der** [comment]
@kc5nra Thanks for the review! Open Source is great

**Pull request merged**